test/update_dev_build.sql

```diff
@@ -0,0 +1,84 @@
+-- adding  accounts,  for suppliers
+
+INSERT INTO account (type_id, enterprise_id, number, parent, label)
+VALUES (2, 1, 40111002 ,0, "SUPPLIER'S ACCOUNT 1"),
+        (2, 1, 40111003 ,0, "SUPPLIER'S ACCOUNT 2");
+
+
+-- creditor group
+INSERT INTO creditor_group (enterprise_id, uuid,name, account_id, locked)
+VALUES (1,'cgsp1', 'Creditor groud supplier1', 286,0),
+        (1, 'cgsp2', 'Creditor groud supplier2', 287,0);
+
+-- creating creditors
+INSERT INTO creditor (uuid, group_uuid,text)
+VALUES ('credtSp1', HUID(BUID('cgsp1')), 'Creditor supplier 1'),
+        ('credtSp2', HUID(BUID('cgsp2')), 'Creditor supplier 2');
+
+-- creating supliers
+
+INSERT INTO supplier (uuid, reference, creditor_uuid, display_name, international,locked)
+VALUES ('supplier1', 0,  HUID( BUID('credtSp1')), 'supplier 1', 1,0),
```

Comment:
The string parameter for BUID() function must be in a `uuid` string format `e2da4b1a-19c7-4903-8399-89ec481ead2c`.

You can generate this string with the function UUID();